MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1925,7 +1925,7 @@ public bool CanStreamCopyAudio(EncodingJobInfo state, MediaStream audioStream, I
                 }
             }
 
-            return bitrate;
+            return Math.Min(bitrate ?? 0, int.MaxValue / 2);
```

Comment:
Shouldn't you still return null, or change the function signature?

Reply:
Done.